walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -1,25 +1,99 @@
-#content {
+input {
+  border-bottom: solid 1px;
+  border-left: none;
+  border-right: none;
+  border-top: none;
+  height: 30px;
+  outline: none;
+  width: 50%;
+}
+
+button {
+  height: 60px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+  width: 50%;
+}
+
+img {
+  height: auto;
+  min-height: 300px;
+  min-width: 300px;
+  width: 100%;
+}
+
+.header-container {
+  align-items: center;
+  display: flex;
+  height: 100px;
+  justify-content: space-evenly;
+  margin-right: auto;
+  margin-left: auto;
+  max-width: 1000px;
+  width: 100%;
+}
+
+.header-item {
+  color: black;
+  font-size: 23px;
+  text-decoration: none;
+}
+
+.content-container {
+  display: flex;
+  flex-direction: column;
+  max-width: 1000px;
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 100%;
 }
 
-.image-thumbnails {
+.about-container {
   display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 100%;
 }
 
-.image-thumbnails img {
-  margin: 10px;
-  max-height: 200px;
-  max-width: 200px;
+.description p {
+  font-size: 19px;
 }
 
-.image-thumbnails img:hover {
-  border: thick solid black;
+.footer-container {
+  display: flex;
+  height: 200px;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 1000px;
+  width: 100%;
+}
+
+.column1 {
+  align-items: left;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  width: 20%;
 }
 
-#random-image-container img {
-  margin-top: 20px;
+.column1 a {
+  color: black;
+  font-size: 20px;
+  margin-left: 100px;
+  text-decoration: none;
 }
+
+.column1 a:hover {
+  text-decoration: underline;
+}
+
+.column2 {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  margin-left: 100px;
+  width: 80%;
+}
+
+/* @media screen and (min-width: 600px) {} */
```

Comment:
nit: should this be cleaned up?

Reply:
just keeping it there for future work to do